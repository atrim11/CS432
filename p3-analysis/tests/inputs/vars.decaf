int x;
int y;
int z;
bool a;
bool b;
bool c;

def int something() {
    return c + b;
}

def void now(int a, int a, int a, int a, int b, int b, int b, void x, void c) {}

void a;