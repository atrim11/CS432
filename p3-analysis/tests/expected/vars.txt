Void variable 'a' on line 14
Type mismatch: int expected but bool found on line 9
Type mismatch: int expected but bool found on line 9
Duplicate symbols named 'a' in scope started on line 12
Duplicate symbols named 'b' in scope started on line 12
Duplicate symbols named 'a' in scope started on line 1
Program does not contain a 'main' function
